### DISCUSSION TITLE
undefined reference to `lv_demo_widgets

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [rzeldent/esp32-smartdisplay#115] [Q&A]
Hi. i still playing around. i got working you esp32-smartdisplay_demo working but the display image builds up slowly (running n the 7" version). So i want to try the LVGL demo but i can't run `lvgl_demo_widgets`
I activated `#define LV_USE_DEMO_WIDGETS 1` in `lv_conf.h`.
i included `demos/widgets/lv_demo_widgets.h`.

I tried alot. added explicit paths for inludes files  to plattformio.ini. tried with copys of demo files in serveral folders. 
I added compiler warnings to the parts of inlcude files to see if the files are procceded/inlcuded.
All inlcuded fine.

But i got always this error:
`undefined reference to `lv_demo_widgets'`

if i comment out the `lv_demo_widgets();` all compile fine.

I am searching for hours an become crazy.
Thank you

**Comment by rzeldent** [comment]
Hi Chris,

I think this requires also compiling the source code in LVGL for the demo.
